### PR TITLE
Set version in one place, bump version for metrics release

### PR DESCRIPTION
**Pull request by pselle**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -27,4 +37,4 @@
       tests_require=[
           'pytest'
       ]
-     )
+      )
```
Comment (ewindisch) [inline setup.py]: Too much indentation on this bracket?

File: setup.py [hunk 2]
```diff
@@ -7,8 +8,17 @@
 install_reqs = parse_requirements('./requirements.txt', session=PipSession())
 reqs = [str(ir.req) for ir in install_reqs]
 
+
+with open('iopipe/iopipe.py', 'r') as fd:
```
Comment (ewindisch) [inline setup.py]: what about just importing iopipe.iopipe and looking at VERSION?

File: setup.py (hunk 1 above)
Comment (pselle) [inline setup.py]: It made flake8 stop complaining (it wasn't failing, but it removed a warning). We can change back _shrug_ I like the previous way it was better personally

Comment (ewindisch) [inline setup.py]: if flake8 wasn't happy and is now, then it's probably right. Could be github formatting making this hard for me to see.

Comment (pselle) [inline setup.py]: No, not just you, it's in the same vertical line as the array

Comment (ewindisch) [inline setup.py]: flake8 is right even when it's wrong. 😭

**Pull request merged**